crypto/params_dup.c

```diff
@@ -11,7 +11,7 @@
 #include <openssl/params.h>
 #include <openssl/param_build.h>
 #include "internal/param_build_set.h"
-#include "internal/e_os.h" /* strcasecmp */
+#include "internal/e_os.h" /* OPENSSL_strcasecmp */
```

Comment:
Wrong change - should be deleted including the #include.

Reply:
Done, thanks